includes/class-newspack-ads-model.php

```diff
@@ -263,10 +264,10 @@ public static function sanitise_ad_unit( $ad_unit ) {
 		}
 
 		$sanitised_ad_unit = array(
-			'name'            => \esc_html( $ad_unit['name'] ),
-			self::AD_CODE     => $ad_unit[ self::AD_CODE ], // esc_js( $ad_unit['code'] ), @todo If a `script` tag goes here, esc_js is the wrong function to use.
-			self::AMP_AD_CODE => $ad_unit[ self::AMP_AD_CODE ], // esc_js( $ad_unit['code'] ), @todo If a `script` tag goes here, esc_js is the wrong function to use.
-			self::AD_SERVICE  => $ad_unit[ self::AD_SERVICE ],
+			'name'           => \esc_html( $ad_unit['name'] ),
+			self::CODE       => $ad_unit[ self::CODE ],
+			self::SIZES      => $ad_unit[ self::SIZES ],
```

Comment:
The code and sizes should be sanitized to some valid, reasonable value here (alphanumeric and `_` allowed for the code, and integer for the sizes). I wasn't able to hack a site in my testing (but it might be possible with the right string for someone more clever/better at JS than me). Currently inputting a width of e.g. `//` will cause havoc with the generated script tag in the header.

Reply:
Size (and ad service type) sanitization in: https://github.com/Automattic/newspack-ads/pull/15/commits/4be47b8bb10476ce7ef116bfda0bae854a1829ce

Sanitizing the Code with `sanitize_title` in https://github.com/Automattic/newspack-ads/pull/15/commits/f6287fcbe7d9a8ea140bd198f3c293738ea45604